doc/changelog.md

```diff
@@ -2,66 +2,133 @@
 
 <h3>New features</h3>
 
-* JAX-compatible functions which run on classical accelerators such as GPUs via `catalyst.accelerate` now support autodifferentiation.
+* JAX-compatible functions that run on classical accelerators, such as GPUs via `catalyst.accelerate`, now support autodifferentiation.
```

Comment:
Seems this one was dropped (but let me know if the current version is intended, it just reads weird to me)
```suggestion
* JAX-compatible functions that run on classical accelerators, such as GPUs, via `catalyst.accelerate` now support autodifferentiation.
```